apps/Oomox Theme Designer/description

```diff
@@ -0,0 +1,8 @@
+A tool to customize icons, xrdb and GTK themes. 
+
+Graphical application for generating different color variations of Oomox (Numix-based) and Materia (ex-Flat-Plat) themes (GTK2, GTK3, Cinnamon, GNOME, Openbox, Xfwm), Archdroid, Gnome-Color, Numix, Papirus and Suru++ icon themes. 
+Have a hack for HiDPI in gtk2.
```

Comment:
I'm not sure 
```suggestion
Has a hack for HiDPI in gtk2.
```

Reply:
I just copy from its [Github](https://github.com/themix-project/oomox#themix-gui-designer).

Reply:
what do you say @Botspot?

Reply:
I'll remove that line altogether - Raspberry Pi has its own way for handling high-res screens - pixel doubling.